.github/CODEOWNERS

```diff
@@ -177,8 +179,8 @@
 
 /tensorflow_addons/seq2seq/ @qlzh727 @guillaumekln
 
-/tensorflow_addons/text/crf.py @squadrick
-/tensorflow_addons/text/tests/crf_test.py @squadrick
+/tensorflow_addons/text/crf.py @squadrick @howl-anderson
+/tensorflow_addons/text/tests/crf_test.py @squadrick @howl-anderson
```

Comment:
Thanks! Mind just removing squadrick

Reply:
OK, I have updated the codeowner file for this.